Cython/Tests/TestScanning.py

```diff
@@ -0,0 +1,104 @@
+from __future__ import unicode_literals
+
+import unittest
+from io import StringIO
+
+from Cython.Compiler import Scanning
+from Cython.Compiler.Symtab import ModuleScope
+from Cython.Compiler.TreeFragment import StringParseContext
+from Cython.Compiler.Errors import init_thread
+
+# generate some fake code - just a bunch of lines of the form "a0 a1 ..."
+code = []
+for ch in range(ord("a"), ord("z")):
+    line = " ".join(["%s%s" % (chr(ch), n) for n in range(10)])
```

Comment:
There's the `strings` module in the stdlib.
```suggestion
for ch in strings.ascii_letters:
    line = " ".join(["%s%s" % (ch, n) for n in range(10)])
```